entry/src/main/js/default/pages/index/index.hml

```diff
@@ -0,0 +1,57 @@
+<!--/*-->
+<!-- * Copyright (C) 2022 Application Library Engineering Group-->
+<!-- *-->
+<!-- * Licensed under the Apache License, Version 2.0 (the "License");-->
```

Comment:
copyright is not required in hml or css files, please remove

Reply:
Sure